tests/test_sanity.py

```diff
@@ -110,6 +112,8 @@ def test_python_ebpf(
         assert_function_in_collapsed(
             "_PyEval_EvalFrameDefault_[pn]", process_collapsed, True
         )  # ensure native user stacks exist
+        assert_function_in_collapsed("setuptools-59.4.0", process_collapsed)  # ensure package info is presented
+        assert_function_in_collapsed("standard-library-3.6.15", process_collapsed)  # ensure Python version is presented
```

Comment:
How can we assure python is being run when the application is executable is `3.6.15`?

Reply:
I just wanted to see if by any chance it'll work just like that because I saw we always use the same container image in the tests. I need to fix this as we discussed.